nixos/modules/services/cluster/dynomite.nix

```diff
@@ -0,0 +1,60 @@
+{ config, pkgs, lib, ... }: with lib; let
+  cfg = config.services.dynomite;
+  configFile = pkgs.writeText "dynomite.yaml" "${cfg.config}";
+in {
+  options = {
+    services.dynomite = {
+      enable = mkEnableOption "dynomite server";
+      config = mkOption {
+        default = "";
+        example = ''
+          dyn_o_mite:
+            listen: 127.0.0.1:8102
+            dyn_listen: 127.0.0.1:8101
+            tokens: '101134286'
+            servers:
+            - 127.0.0.1:6379:1
+            data_store: 0
+            mbuf_size: 16384
+            max_msgs: 300000
+        '';
+        type = types.lines;
+        description = "Verbatim dynomite configuration to use";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    assertions = [{
+      assertion = cfg.config != "";
+      message = "You must provide dynomite.config.";
+    }];
+
+    services.redis.enable = true;
+    systemd.services.dynomite = {
+      description = "Dynomite";
+      wantedBy = [ "multi-user.target" ];
+      after = ["network.target" "redis.service" ];
+      requires = [ "redis.service" ];
+
+      serviceConfig = {
```

Comment:
Without the use of `DynamicUser` or `User` and `Group` defined this will run as `root`.